explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -711,8 +715,31 @@ topsortTxsOrFail f =
 
 -- | Deserialize Cardano or RSCoin address and convert it to Cardano address.
 -- Throw exception on failure.
-cAddrToAddr :: MonadThrow m => CAddress -> m Address
-cAddrToAddr cAddr@(CAddress rawAddrText) =
+-- cAddrToAddr :: MonadThrow m => CAddress -> m Address
```

Comment:
@ksaric what's the point of having these lines left in comments ?

Reply:
@akegalj https://github.com/input-output-hk/cardano-sl/pull/1646#discussion_r140972742